WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditAddressForm.swift

```diff
@@ -90,8 +90,13 @@ extension EditAddressHostingController: UIAdaptivePresentationControllerDelegate
     func presentationControllerDidAttemptToDismiss(_ presentationController: UIPresentationController) {
         UIAlertController.presentDiscardChangesActionSheet(viewController: self) { [weak self] in
             self?.dismiss(animated: true)
+            self?.rootView.viewModel.userDidCancelFlow()
         }
     }
+
+    func presentationControllerDidDismiss(_ presentationController: UIPresentationController) {
+        rootView.viewModel.userDidCancelFlow()
+    }
```

Comment:
Q: since we have `DidAttemptToDismiss` handled, is there any case where `DidDismiss` called?

Reply:
When `presentationControllerShouldDismiss` returns `true` then `presentationControllerDidDismiss` is called.

Reply:
Right, I forgot about that case 😅